test/e2e/haproxy_timeouts_test.go

```diff
@@ -141,12 +142,12 @@ func TestHAProxyTimeouts(t *testing.T) {
 		cmd := []string{
 			"grep",
 			"-oP",
-			"timeout\\s+" + timeout.Name + "\\s*\\K[0-9]+(?:us|ms|s|m|h|d)",
+			"timeout\\s+" + timeout.Name + "\\s*\\K" + timeout.Value,
```

Comment:
Given that we are changing the RE in this commit the second whitespace (i.e., \s*) should be \s+.